dashboard/test/controllers/api/v1/projects/public_gallery_controller_test.rb

```diff
@@ -88,62 +42,44 @@ class Api::V1::Projects::PublicGalleryControllerTest < ActionController::TestCas
     response: :bad_request,
   )
 
-  test 'project details are correct listing published applab projects' do
+  test 'project details are correct listing all published featured projects' do
     Rails.stubs(:env).returns(ActiveSupport::StringInquirer.new('production'))
-    get :index, params: {project_type: 'applab', limit: 1}
+    get :index, params: {project_type: 'all', limit: 1}
 
     assert_response :success
     assert_equal "max-age=5, public", @response.headers["Cache-Control"]
     categories_list = JSON.parse(@response.body)
-    assert_equal 1, categories_list.length
+    assert_equal 11, categories_list.length
+    assert_empty categories_list['gamelab']
+    assert_empty categories_list['playlab']
+    assert_empty categories_list['artist']
     project_row = categories_list['applab'].first
-    assert_equal 'Charlies App', project_row['name']
-    assert_equal storage_encrypt_channel_id(22, 33), project_row['channel']
-    assert_equal '/v3/files-public/charlies_thumbnail.png', project_row['thumbnailUrl']
+    assert_equal 'Applab1_to_be_featured', project_row['name']
+    assert_equal 22, project_row['storage_id']
+    assert_equal 33, project_row['id']
+    assert_equal '/v3/files/thumbnail.png', project_row['thumbnailUrl']
     assert_equal 'applab', project_row['type']
-    assert_equal '2017-03-03T00:00:00.000-08:00', project_row['publishedAt']
-    assert_equal '13+', project_row['studentAgeRange']
-    assert_equal 'P', project_row['studentName']
-  end
-
-  test 'project details are correct without thumbnail' do
```

Comment:
Replaced this test with a new one in [projects_list_test.rb](https://github.com/code-dot-org/code-dot-org/blob/414ddba7b9723f8f7f4ab6c0a5b4fe65b2503688/dashboard/test/lib/projects_list_test.rb#L246-L268).